documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -1,82 +1,55 @@
 [[marshalling]]
 = Marshalling
-Marshalling is the process of converting Java POJOs into something that can be written in a format that can be transferred over the wire.
-Unmarshalling is the reverse process whereby data read from a wire format is transformed back into Java POJOs. {brandname} uses marshalling/unmarshalling in order to:
+
+Marshalling is the process of converting Java objects into something that can be written in a format that can be transferred over the wire.
+Unmarshalling is the reverse process whereby data read from a wire format is transformed back into Java objects. {brandname} uses marshalling/unmarshalling in order to:
 
 * Transform data so that it can be send over to other {brandname} nodes in a cluster.
 * Transform data so that it can be stored in underlying cache stores.
 * Store data in {brandname} in a wire format to provide lazy deserialization capabilities.
 
-== The Role Of JBoss Marshalling
-Since performance is a very important factor in this process, {brandname} uses JBoss Marshalling framework instead of standard Java Serialization in order to marshall/unmarshall Java POJOs. Amongst other things, this framework enables {brandname} to provide highly efficient ways to marshall internal {brandname} Java POJOs that are constantly used. Apart from providing more efficient ways to marshall Java POJOs, including internal Java classes, JBoss Marshalling uses highly performant `java.io.ObjectOutput` and `java.io.ObjectInput` implementations compared to standard `java.io.ObjectOutputStream` and `java.io.ObjectInputStream`.
-
-== Support For Non-Serializable Objects
-From a users perspective, a very common concern is whether {brandname} supports storing non-Serializable objects. In 4.0, an {brandname} cache instance can only store non-Serializable key or value objects if, and only if:
+{brandname} handles the marshalling of all it's internal types, so users only have to be concerned with the marshalling
```

Comment:
Global comment: "its" for possessive 

usually better to avoid possessive in reference to Infinispan because it's kind of anthropomorphic. in this case, just drop and change to "{brandname} handles marshalling for all internal types. Users need to be concerned with marshalling of ..."